src/test/scala/akka/contrib/process/BlockingProcessSpec.scala

```diff
@@ -20,36 +21,37 @@ class BlockingProcessSpec extends WordSpec with Matchers with BeforeAndAfterAll
 
   implicit val system = ActorSystem("test", testConfig)
 
-  implicit val processCreationTimeout = Timeout(2.seconds)
+  implicit val processCreationTimeout = Timeout(2.seconds.dilated)
 
   "A BlockingProcess" should {
     "read from stdin and write to stdout" in {
       val command = getClass.getResource("/echo.sh").getFile
       new File(command).setExecutable(true)
 
-      val probe = TestProbe()
+      val streamProbe = TestProbe()
+      val exitProbe = TestProbe()
       val stdinInput = List("abcd", "1234", "quit")
-      val receiver = system.actorOf(Props(new Receiver(probe.ref, command, stdinInput, 1)), "receiver1")
+      val receiver = system.actorOf(Props(new Receiver(streamProbe.ref, exitProbe.ref, command, stdinInput, 1)), "receiver1")
```

Comment:
Introduced the distinction between stream and exit probes as it is technically possible to get an exit prior to the stream data